spring-cloud-gcp-data-firestore/src/main/java/org/springframework/cloud/gcp/data/firestore/repository/query/PartTreeFirestoreQuery.java

```diff
@@ -69,6 +71,8 @@
 					.put(GREATER_THAN, StructuredQuery.FieldFilter.Operator.GREATER_THAN)
 					.put(LESS_THAN_EQUAL, StructuredQuery.FieldFilter.Operator.LESS_THAN_OR_EQUAL)
 					.put(LESS_THAN, StructuredQuery.FieldFilter.Operator.LESS_THAN)
+					.put(IN, StructuredQuery.FieldFilter.Operator.IN)
+					.put(CONTAINING, StructuredQuery.FieldFilter.Operator.ARRAY_CONTAINS_ANY)
```

Comment:
Firestore has both `contains` and `contains-any`. Why are you using `contains-any` here vs `contains`?
Also, how can we support both types of contains?

Reply:
`contains-any` with a one element list is exactly the same as `contains`, so I think we don't need to support `contains` separately

Reply:
What about with a list of more than one element?

Reply:
`contains` supports only singular values and `contains-any` combines `contain` filters using `OR`

Reply:
There's this caveat in the [documentation](https://cloud.google.com/firestore/docs/query-data/queries):

> You can combine `array-contains` with `in` but not with `array-contains-any`.